Makefile

```diff
@@ -60,17 +60,14 @@ alldeps:
 	docker-compose -p athensdev up -d minio
 	docker-compose -p athensdev up -d jaeger
 	echo "sleeping for a bit to wait for the DB to come up"
-	sleep 5	
+	sleep 5
 
 .PHONY: dev
 dev:
+	docker-compose -p athensdev up -d minio
```

Comment:
so now the only thing differing alldeps and dev is jeager? does it make sense

Reply:
Not really, but you can't run unit tests without minio so maybe we should just rename `alldeps` to `dev` and remove the old `dev`.

Reply:
it makes sense to me as well, @marwan-at-work had some strong opinions against it

Reply:
Actually now that I think about it, we don't even need redis in dev 😄 -- which we can replace with minio, but from the slack discussion it seemed that peeps are happy making minio not a dependency of tests and therefore just mongo in `dev` would be enough in the future. We can add minio here as a temporary thing until tests are cleaned up. 

The only reason I want dev to be minimal is for *fast* iteration on hitting cacheFill every single time. In otherwords, cleaning the storage which we can achieve in other ways (would be more efficient)

Reply:
Let's revisit when we get around to fixing up tests, now that our Docker builds are moving.